content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -11,7 +11,7 @@ tocDepth: 2
 
 <Admonition type="warning">
 
-Support for views is currently a very early preview feature. You can add a view to your Prisma schema with the `view` keyword, but you cannot introspect your schema with `db pull` or apply your schema to your database with Prisma Migrate and `db push`.<br /><br />For updates on progress with this feature, follow [our GitHub issue](https://github.com/prisma/prisma/issues/17335).
+Support for views is currently a very early [Preview](/about/prisma/releases#preview) feature. You can add a view to your Prisma schema with the `view` keyword or introspect the views in your schema with `db pull`.<br /><br />For updates on progress with this feature, follow [our GitHub issue](https://github.com/prisma/prisma/issues/17335).
```

Comment:
This now lost the info that you can not use Prisma MIgrate or `db push` fully yet.